src/com/itacademy/eshop/product/types/author.java

```diff
@@ -0,0 +1,13 @@
+package com.itacademy.eshop.product.types;
+
+public class author {
```

Comment:
Must use Upper camel case for classes, records, interfaces, annotations and enums